esp/bindings/SOAP/xpp/sxt/XmlTokenizer.h

```diff
@@ -222,7 +222,7 @@ namespace sxt {
    *
    */
    //int XmlTokenizer::next() throw(XmlTokenizerException) {
-   int next() throw(XmlTokenizerException) {
+   int next() {
```

Comment:
throw specifiers are deprecated in c++11

Reply:
I need this when enable "std=c++1z" otherwise just warning.